### ISSUE TITLE
ipmitool segfaults on macOS

### ISSUE THREAD

**Issue by bahamat** [TritonDataCenter/pkgsrc#79]

**Comment by mamash** [comment]
Hm I probably don't have a way to test this, but just executing ipmitool or getting help works for me.

**Comment by bahamat** [comment]
I figured out that this happens after prompting for password.

```
$ ipmitool -I lanplus -H 127.0.0.1
Password: 
zsh: segmentation fault  ipmitool -I lanplus -H 127.0.0.1
```

If there's no password prompt, then no segfault.

```
$ ipmitool -I lanplus -H 127.0.0.1 -P abc
^C
SIGN INT: Close Interface IPMI v2.0 RMCP+ LAN Interface
```

Note, I don't have anything capable of servicing IPMI packets on 127.0.0.1.

**Comment by jperkin** [comment]
I pushed a fix for this, look out for ipmitool-1.8.18nb1 in the next few days.
